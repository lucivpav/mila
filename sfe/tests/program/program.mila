program test;
begin
  writeln(1);
end.
---input---
begin
  writeln(1);
end.
program test;
---input---
program test;
program test2;
begin
  writeln(1);
end.
