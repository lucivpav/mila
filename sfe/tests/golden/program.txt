1
0
---output---
1
0
---output---
256
